src/store.js

```diff
@@ -8,7 +8,7 @@ export default new Vuex.Store({
     barColor: 'rgba(0, 0, 0, .8), rgba(0, 0, 0, .8)',
     barImage: 'https://demos.creative-tim.com/material-dashboard/assets/img/sidebar-1.jpg',
     drawer: null,
-    url: 'http://127.0.0.1:5000',
+    url: 'http://34.76.18.17:5000',
```

Comment:
this change is not necessary. You can have the test url but don't commit it into github.